JavaLearning-master/src/operatorler-main/Comparator.java

```diff
@@ -1,6 +1,6 @@
 package operatorler;
 
-public class IliskiselKarsilastirmaMantiksalOparatorler {
+public class Comparator {

```

Comment:
Yusuf this class looks batter but it is not could you reffaktry class name as operatorComperator

Reply:
Thanks Serhat, great idea,
I am going to update it